src/plugins/plot/chart/MctChart.vue

```diff
@@ -286,20 +296,25 @@ export default {
       // and we need to use the Open MCT root element as the root of the intersection observer.
       if (entry.target === this.chartContainer) {
         const wasVisible = this.chartVisible;
-        this.chartVisible = entry.isIntersecting;
-        if (!this.chartVisible) {
-          // destroy the chart
+        const isNowVisible = entry.isIntersecting;
+        const chartInOverlayWindow = this.chartContainer?.closest('.js-overlay') !== null;
```

Comment:
intersection observers in previews behave badly as the overlay covers our root element! try to detect that we're in one.